src/adagrams.js

```diff
@@ -1,15 +1,107 @@
+'use strict';
+
+import { letterPool, scoreChart } from './constants.js';
+
+
+const createLetterPoolList = (letterPoolDict) => {
+  //Function has one parameter, a dictionary, and returns a list of letters.
+  let letterPoolList = [];
+
+  for (const [letter, freq] of Object.entries(letterPoolDict)) {
+    for (let i = 0; i < freq; i++) {
+      letterPoolList.push(letter);
+    }
+  }
```

Comment:
Something extra to think about is how you could implement this same functionality with the `...` syntax that is provided and if you think it provides any advantages against what you have.